lib/collections/addon/components/discover-page/active-filters/filter/template.hbs

```diff
@@ -8,6 +8,7 @@
         <span aria-label={{@ariaLabel}}>
             <button
                 data-test-active-filter-remove-button
+                type='button'
                 class='fa fa-times-circle'
                 local-class='removeActiveFilter'
                 {{action this.removeFilterItem}}
```

Comment:
Could use element modifier here as well. But since you're not touching this line, probably not very important now.